### DISCUSSION TITLE
osu!mania late/early note below perfect/great/etc

### DISCUSSION THREAD

**Discussion by Sarisan** [ppy/osu#23172] [Ideas]
Would be great to have an option that enables such functionality, like when I see perfect or great or etc playing mania

```
  PERFECT
0.1 ms late

   GREAT
0.5 ms early
```

Something like that

**Comment by peppy** [comment]
We have intentions of showing early/late hits visually, but we would never show the raw number like that. It's too busy and not useful when you're playing the game.
